go.mod

```diff
@@ -16,17 +16,19 @@ require (
 	github.com/klauspost/compress v1.7.5 // indirect
 	github.com/klauspost/cpuid v1.2.1 // indirect
 	github.com/klauspost/pgzip v1.2.1 // indirect
+	github.com/mailru/easyjson v0.7.0 // indirect
 	github.com/operator-framework/operator-sdk v0.10.0
 	github.com/prometheus/client_golang v1.4.1
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
+	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9 // indirect
 	gopkg.in/yaml.v2 v2.2.8
 	k8s.io/api v0.0.0-20190612125737-db0771252981
 	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
 	k8s.io/apiserver v0.0.0-20190228174905-79427f02047f // indirect
 	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/code-generator v0.17.3 // indirect
+	k8s.io/gengo v0.0.0-20190822140433-26a664648505 // indirect
```

Comment:
This PR shouldn't need to change deps, right? Can you try reverting `go.*` and just run `make build` without any manual `go mod` commands?

Reply:
Build runs fine with reverting the go.sum and go.mod files.